src/tests/ui/planner/workitem-detailpage.ts

```diff
@@ -0,0 +1,13 @@
+import { ElementFinder, $ } from 'protractor';
+import { WorkItem } from './index';
```

Comment:
We can remove this as well. I don't see where it's being used.

Reply:
done!